OpenKeychain/src/main/java/org/sufficientlysecure/keychain/provider/KeychainDatabase.java

```diff
@@ -306,6 +314,11 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     return;
                 }
             */
+            case 20:
+                db.execSQL(CREATE_OVERRIDDEN_WARNINGS);
+                if (oldVersion == 20) {
+                    return;
+                }
```

Comment:
currently, (As far as I understand) if someone is upgrading from 18 or 19 to 21, your SQL will never be executed. We really need to fix our upgrade method...